components/Droplet.js

```diff
@@ -273,8 +273,10 @@
             $Ember.merge(options, this.get('options'));
             set(this, 'options', options);
 
-            this.DropletEventBus && this.DropletEventBus.subscribe(EVENT_NAME, this, (...files) => {
-                this.send('prepareFiles', ...files);
+            this.DropletEventBus && this.DropletEventBus.subscribe(EVENT_NAME, this, (ctx, ...files) => {
```

Comment:
Will the `ctx` argument always be provided on publishing events? Or is it possible currently that it will contain the first file argument?